nemo/collections/tts/modules/common.py

```diff
@@ -628,6 +628,9 @@ def __init__(
                 kernel_size=kernel_size,
                 use_partial_padding=use_partial_padding,
             )
+        else:
+            raise Exception("{} affine model not supported".format(affine_model))
```

Comment:
We mostly use f-string, i.e. `f"Affine model is not supported: {affine_model}. Please choose either 'wavenet' or 'simple_conv' instead."`.

Besides, This is a `ValueError`. Could you please replace `Exception` with `ValueError`? Thanks.